src/components/molecules/Footer/index.jsx

```diff
@@ -19,7 +20,9 @@ import "./footer.scss";
 
```

Comment:
This line and the line above "//> CSS" (cannot be marked) should be removed.
```suggestion
```

Reply:
We split imports per header and a new line.